themes/README.md

```diff
@@ -0,0 +1,62 @@
+## Available Themes
+
+With inbuilt themes you can customize the look of the card without doing any manual customization.
+
+Use `?theme=THEME_NAME` parameter like so :-
+
+```md
+![Anurag's github stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&theme=dark&show_icons=true
+```
+
+## Stats
+
+> These themes work both for the Stats Card and Repo Card.
+
+|                               |                                        |                                              |
+| :---------------------------: | :------------------------------------: | :------------------------------------------: |
+|     `dark` ![dark][dark]      |     `radical` ![radical][radical]      |           `merko` ![merko][merko]            |
+| `gruvbox` ![gruvbox][gruvbox] | `tokyonight` ![tokyonight][tokyonight] |        `onedark` ![onedark][onedark]         |
+|  `cobalt` ![cobalt][cobalt]   |  `synthwave` ![synthwave][synthwave]   | `highcontrast` ![highcontrast][highcontrast] |
+| `dracula` ![dracula][dracula] |     `default` ![default][default]      |         [Add your theme][add-theme]          |
+
+## Repo Card
+
+> These themes work both for the Stats Card and Repo Card.
+
+|                                        |                                                          |                                                       |
+| :------------------------------------: | :------------------------------------------------------: | :---------------------------------------------------: |
+| `dark` ![dark_repocard][dark_repocard] |          `radical` ![radical][radical_repocard]          |           `merko` ![merko][merko_repocard]            |
+| `gruvbox` ![gruvbox][gruvbox_repocard] |     `tokyonight` ![tokyonight][tokyonight_repocard]      |        `onedark` ![onedark][onedark_repocard]         |
+|  `cobalt` ![cobalt][cobalt_repocard]   |       `synthwave` ![synthwave][synthwave_repocard]       | `highcontrast` ![highcontrast][highcontrast_repocard] |
+| `dracula` ![dracula][dracula_repocard] | `default_repocard` ![default_repocard][default_repocard] |              [Add your theme][add-theme]              |
```

Comment:
Change the add your theme link to this, that would be better i guess since it will directly open the file to edit

https://github.com/anuraghazra/github-readme-stats/edit/master/themes/index.js

Reply:
Ohh yeah, I forgot about you could do that.